pkg/plugins/golang/v4/plugin.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v4
+
+import (
+	"sigs.k8s.io/kubebuilder/v3/pkg/config"
+	"sigs.k8s.io/kubebuilder/v3/pkg/model/stage"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang"
+)
+
+const pluginName = "base." + golang.DefaultNameQualifier
+
+var (
+	pluginVersion            = plugin.Version{Number: 4}
+	supportedProjectVersions = []config.Version{{Number: 4, Stage: stage.Alpha}}
+)
```

Comment:
It shows the reason for the issue. 
The plugin version is : 4 alpha
The Project Version ( Project file config ) : would be the same of go/v3 plugin -> Plugin Config version 3